src/structures/ClientUser.js

```diff
@@ -25,8 +25,8 @@ class ClientUser extends User {
        * @type {?boolean}
        */
       this.mfaEnabled = typeof data.mfa_enabled === 'boolean' ? data.mfa_enabled : null;
-    } else if (typeof this.mfaEnabled === 'undefined') {
-      this.mfaEnabled = null;
+    } else {
+      this.mfaEnabled ??= null;
```

Comment:
i think that this should be left unchanged

Reply:
Why

Reply:
nvm seems like i didn't understand what does the nullish assignment actually do

Reply:
This only assigns the value if the property on the left-hand side is nullable